app/public/index.html

```diff
@@ -462,24 +492,31 @@
 
       if (theme === 'dark') document.querySelector('body').style.backgroundColor = '#24252A'
 
-      let selectedLoader = 'skeleton-login'
+      const mobileTag = screen.width < MOBILE_BREAKPOINT ? '-mobile' : ''
+      let selectedLoader = 'skeleton-login' + mobileTag
       if (/\/wallet\/transfer/.test(redirectPath)) {
-        selectedLoader = 'skeleton-wallet-transfer'
+        selectedLoader = 'skeleton-wallet-transfer' + mobileTag
       } else if (/\/wallet\/topup/.test(redirectPath)) {
-        selectedLoader = 'skeleton-wallet-topup'
+        selectedLoader = 'skeleton-wallet-topup' + mobileTag
       } else if (/\/wallet\/history/.test(redirectPath)) {
-        selectedLoader = 'skeleton-wallet-history'
+        selectedLoader = 'skeleton-wallet-history' + mobileTag
       } else if (/\/wallet\/settings/.test(redirectPath)) {
-        selectedLoader = 'skeleton-wallet-settings'
+        selectedLoader = 'skeleton-wallet-settings' + mobileTag
       } else if (/\/wallet\/home\/collectibles/.test(redirectPath)) {
-        selectedLoader = 'skeleton-wallet-home-collectibles'
+        selectedLoader = 'skeleton-wallet-home-collectibles' + mobileTag
       } else if (/\/wallet\/home/.test(redirectPath)) {
-        selectedLoader = 'skeleton-wallet-home'
+        selectedLoader = 'skeleton-wallet-home' + mobileTag
       } else if (/\/popup/.test(redirectPath) || /\/confirm/.test(redirectPath) || /\/providerchange/.test(redirectPath) || /\/userinforequest/.test(redirectPath)) {
```

Comment:
please add /redirect check here as well. In case service worker is not loaded, it falls back to index.html which loads /redirect from vuex router which will not show the loader here.

Reply:
> please add /redirect check here as well. In case service worker is not loaded, it falls back to index.html which loads /redirect from vuex router which will not show the loader here.

Done